pom.xml

```diff
@@ -339,7 +340,7 @@
       <dependency>
         <groupId>io.netty</groupId>
         <artifactId>netty-all</artifactId>
-        <version>4.1.22.Final</version>
+        <version>4.1.30.Final</version>
```

Comment:
Is this version upgrade required for gRPC?

Reply:
Not required. I was trying to workaround netty hang issue.
However, it'll eventually need to be bumped as there is fix in progress that we'll need to take.